WordPress/Classes/Extensions/NSString+Helpers.swift

```diff
@@ -3,12 +3,18 @@ import Foundation
 
 extension NSString
 {
-    /// Returns the string's hostname. If there's any
+    /// Returns the string's hostname
+    ///
+    /// -   returns: hostname, if any
+    ///
```

Comment:
I believe in simple cases like this the `returns` is redundant as it provides the same information as the summary. It doesn't hurt to have it there but generally I wouldn't bother to add it.